bigbluebutton/2022-10-virtuos-usability-test/README.md

```diff
@@ -0,0 +1,5 @@
+# Usability Test of BigBlueButton
+- Authors: Ammar Almohammad Alaabed, Ibrahem Atma, Sebastian Beyrodt, Stefano Casalaina, Imogen Hüsing, Abdulrahman Sakhnieh, Sonja Seyfried, Johan David Siecke, Katharina Wellmann
+(under the supervision of apl. Prof. Dr. Kai-Christoph Hamborg)
+  - on behalf of the university's center for digital teaching (virtUOS)
```

Comment:
You probably want to add which university you are talking about:

```suggestion
  - on behalf of Osnabrück University's center for digital teaching (virtUOS)
```

Reply:
Oh, sure, I forgot to change that. You can safely delete the .DS_store file too (I just left it in because I was unsure at the moment about whether it is relevant in some way). Thanks!

Reply:
that's handy to know :^)